docs/ops/opset5.md

```diff
@@ -0,0 +1,149 @@
+# Operation Set `opset4` Specification {#openvino_docs_ops_opset4}
```

Comment:
Change `opset4` to `opset5` everywhere in the document.

Reply:
Done.